src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,14 +34,16 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart = shoppingCartService.getByUser(
+                userService.findByEmail(authentication.getName()).get());
```

Comment:
let's throw an exception in case user is not found, the same for other similar cases